wss-unified-agent.config

```diff
@@ -17,7 +17,6 @@ npm.runPreStep=true
 #------------------------------------------------------------------------
 
 productName=Mobile
-projectName=sdk-js
```

Comment:
What is the consequence of leaving out the project name?

Reply:
I can see the projectToken in the same file - I'm assuming this is what you mean by using token instead of project name.

Reply:
discussed on slack. https://github.com/ForgeRock/forgerock-javascript-sdk/runs/6280060640?check_suite_focus=true is what we are fixing with this